ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -2,14 +2,69 @@
 {% if spanning_tree is defined and spanning_tree is not none %}
 ## Spanning Tree Summary
 
-Mode: {{ spanning_tree.mode }}
+STP mode running: **{{ spanning_tree.mode }}**
```

Comment:
Change to `STP mode:`. Running is confusing, since this may or may not be the running mode. (Ex in MLAG)

Reply:
Fixed it, thanks!